### DISCUSSION TITLE
Extension doesn't make change the default browser when opening new tab.

### DISCUSSION THREAD

**Discussion by VR952004** [dailydotdev/daily#1656] [Feature Requests]
I learned about daily.dev on a video by fireship. I really liked the idea behind it and I find it convenient to stay up-to-date with the world of technology. The only request i have is to make it such that it doesn't open my feed whenever I open a new tab. I have been very accustomed to the original google chrome browser page and this is just feeling weird. Instead of doing that could please make it such the extension icon is seen near the search bar (as shown in the image below) and we can click on it whenever I feel like checking tech news.

![Screenshot 2024-12-13 120137](https://github.com/user-attachments/assets/2a5bdfd9-9b7c-4297-8b11-88e147460539)

This is how the browser is looking when the extension is enabled

![image](https://github.com/user-attachments/assets/34fbb3a9-9c36-4ff4-a7b6-d1137ac9e2ed)

This is how the browser is when the extension is disabled

![image](https://github.com/user-attachments/assets/ca77a72b-97a8-402d-9745-666c3898bcc9)

Please make this change.

**Comment by idoshamun** [comment]
Hi @VR952004,

Our browser extension is a new tab extension, so this can't be disabled.
I suggest uninstalling the extension and using our webapp https://app.daily.dev
